app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardController < ApplicationController
+    def index
+        @user = policy_scope(User)
+        @my_interview_ids = @user.interviews.map do |interview|
+        interview.id
+      end
+      @all_bookings = Booking.where(interview_id: @my_interview_ids)
```

Comment:
I believe we have an error in the associations in our user model regarding interviews (I made a comment for this in Philipp's and Nico's branch). I think this is the reason why the dashboard doesn't work. Let's solve this together :)